### ISSUE TITLE
what is dist_mat_loss?

### ISSUE THREAD

**Issue by YiyuHong** [NVIDIA-Digital-Bio/proteina#11]
Hi,

I see there is a [dist_mat_loss](https://github.com/NVIDIA-Digital-Bio/proteina/blob/15e5b5e2935a5f4d81e32dd4987a903eef352bcb/proteinfoundation/proteinflow/proteina.py#L247) that always output nan when I running a motif scaffolding training test with a small dataset. And it seems the default threshold of  `thres_aux_2d_loss=0.6` result in the nan. 
Can it be just ignored? I can't find this loss in the paper.

```        
        # Add mask to only account for pairs that are closer than thr in ground truth
        max_dist = self.cfg_exp.loss.thres_aux_2d_loss
        if max_dist is None:
            max_dist = 1e10
        pair_mask_thr = gt_pair_dists < max_dist  # [*, n, n]
        total_pair_mask = pair_mask * pair_mask_thr  # [*, n, n]

        # Compute loss
        den = torch.sum(total_pair_mask, dim=(-1, -2)) - nres
        dist_mat_loss = torch.sum(
            (gt_pair_dists - pred_pair_dists) ** 2 * total_pair_mask, dim=(-1, -2)
        )  # [*]
        dist_mat_loss = dist_mat_loss / den  # [*]
```

Thanks in advance.

**Comment by YiyuHong** [comment]
There is a bug when training motif scaffolding, in the following line should add `x_1 = ang_to_nm(x_1)`
https://github.com/NVIDIA-Digital-Bio/proteina/blob/15e5b5e2935a5f4d81e32dd4987a903eef352bcb/proteinfoundation/nn/motif_factory.py#L336
